hydra-node/src/Hydra/Options.hs

```diff
@@ -60,6 +78,53 @@ data Options = Options
   }
   deriving (Eq, Show)
 
+defaultOptions :: Options
+defaultOptions =
```

Comment:
Just seeing this next to `Arbitrary Options` below. What do we think of `class Default`?